// tbclient.SetUserBlack.SetUserBlackReqIdl
syntax = "proto3";

import "CommonReq.proto";

message SetUserBlackReqIdl {
    message DataReq {
        CommonReq common = 1;
        int64 black_uid = 2;
        message PermissionList {
            int32 follow = 1;
            int32 interact = 2;
            int32 chat = 3;
        }
        PermissionList perm_list = 3;
    }
    DataReq data = 1;
}
